Makefile

```diff
@@ -44,7 +47,11 @@ manifests: controller-gen
 
 # Run go fmt against code
 fmt:
-	go fmt ./...
+	hack/go-fmt.sh
+
+# Run go lint against code
+lint:
+	golint -set_exit_status -min_confidence 0.3 ./...
```

Comment:
When I ran "make lint" after I performed "go mod vendor", I found that go-lint was running on all files in the /vendor directory too. 
Tried the following locally, and it seemed to work.

      golint -set_exit_status -min_confidence 0.3 $(go list -f '{{ .ImportPath }}' ./...)

Reply:
Good catch. Fixed.